src/azure-cli-core/azure/cli/core/commands/parameters.py

```diff
@@ -249,7 +249,8 @@ def get_location_type(cli_ctx):
         options_list=('--location', '-l'),
         completer=get_location_completion_list,
         type=get_location_name_type(cli_ctx),
-        help="Location. You can configure the default location using `az configure --defaults location=<location>`",
+        help="Location. Run `az account list-locations --output table` to get a list of available location names. "
```

Comment:
Be less verbose and keep this consistent with the normal text:
`Values from: az account list-locations.` 

The right place for this is the `populator-commands` field, but we can't do that because you can't set it programmatically. Still, we should keep it as concise.